src/common/options/global.yaml.in

```diff
@@ -1365,6 +1365,14 @@ options:
   desc: Hexdump message to debug log on message send
   default: false
   with_legacy: true
+- name: ms_inject_drop_mon_forward_msgs
+  type: float
+  level: dev
+  desc: Inject drop of mon forward messages every Nth message if negative, drop all messages
```

Comment:
old description? value cannot be negative?

Reply:
this comment still looks relevant

Reply:
Done